Assets/MixedRealityToolkit-SDK/Features/Input/GazeProvider.cs

```diff
@@ -217,9 +223,32 @@ public override bool TryGetPointerRotation(out Quaternion rotation)
                 rotation = Quaternion.identity;
                 return false;
             }
+
+            #endregion IMixedRealityPointer Implementation
+
+            /// <summary>
+            /// Press this pointer. This sends a pointer down event across the input system.
+            /// </summary>
+            /// <param name="mixedRealityInputAction">The input action that corresponds to the pressed button or axis.</param>
+            /// <param name="handedness">Optional handedness of the source that pressed the pointer.</param>
+            public void PressPointer(MixedRealityInputAction mixedRealityInputAction, Handedness handedness = Handedness.None)
```

Comment:
RaisePointerDown?

Reply:
I left these names from my earlier, more full refactor, where pointers might want to do more than just raise the events when they respond to a "press". For now, I can update them to your suggestions, as they're more descriptive for the current implementation.